app/src/main/res/layout/activity_main.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I'm sorry, i just noticed that I changed the layout to ConstraintLayout.

Reply:
But does it still work as expected?

Reply:
Just saw it. I have changed it back to CoordinatorLayout.